song-server/src/main/java/bio/overture/song/server/controller/AnalysisController.java

```diff
@@ -49,19 +49,6 @@
     description = "Create, Read, Update, publish, unpublish, suppress and search analyses")
 public class AnalysisController {
 
-  private static final String EXAMPLE_ANALYSIS_INFO_JSON =
-      "{\n"
-          + "  \"info\" : {\n"
-          + "    \"extra_donor_info\" : {\n"
-          + "      \"physical\" : {\n"
-          + "        \"eye_colour\" : \"blue\",\n"
-          + "        \"hair_colour\" : \"brown\"\n"
-          + "      },\n"
-          + "      \"occupation\" : \"engineer\"\n"
-          + "    }\n"
-          + "  }\n"
-          + "}";
```

Comment:
Weird that this was here, looks like it is never used. Thanks for cleaning it up.